src/assets/styles/_banner.scss

```diff
@@ -1,25 +1,36 @@
 $block-class: 'banner';
+@import 'colors';
 
 @at-root {
     .#{$block-class} {
         text-align: left;
-        margin: 0 1rem 0 1rem;
-        color: #455A64;
+        margin: 0 1rem 1rem 0;
 
         & &__header {
-            margin: 1rem 0;
+            margin-bottom: 0.5rem;
             font-size: 3rem;
             font-weight: 600;
+            color: $blue-grey-darken-2;
         }
 
-        & &__update-time {
+        & &__update-box {
+            display: flex;
+            margin-left: 0.2rem;
+        }
+
+        & &__update-label {
             font-weight: 600;
-            font-size: 1.25rem;
+            font-size: 1rem;
             margin-bottom: 0.25rem;
+            color: $blue-grey;
         }
 
-        & &__description {
-            font-size: 1.25rem;
+        & &__update-time {
+            font-weight: 500;
+            font-size: 1rem;
+            margin-bottom: 0.25rem;
+            margin-left: 0.25rem;
```

Comment:
So this is fine but a good tip for the future is that you can actually do all `margin` or `padding` on one line with multiple arguments. See https://www.w3schools.com/css/css_margin.asp for details.

For four arguments, the order is TRBL, so it would look like... `margin: 0 0 0.25rem 0.25rem;` 
There are also options for two or three arguments. If you provide two arguments, the first applies to top and bottom and the second to left-right. Like, `margin: 0 1rem;` If you provide three arguments, it's T, LR, then B. `margin: 0 1rem 0.5rem;`